### ISSUE TITLE
使用BISRNRT作为Net训练模型时出现问题

### ISSUE THREAD

**Issue by Gchang9** [ggsDing/Bi-SRNet#7]
您好，在使用SSCDl作为Net训练时，没有问题。在使用BiSRNet作为Net训练时，出现了问题，参数和数据及数据读取方式全部是按照您的代码做的。
出现的问题是：
训练的时候显示：WARNING:Nan or Inf found in Input tensor，且train_seg loss bn_loss均为nan

**Comment by Gchang9** [comment]
似乎关闭Tensorboard就不会出现这个问题，请问是为什么呢？

**Comment by Gchang9** [comment]
关闭Tensorboard之后不会报warning，但训练几个epoch之后还是会出现train_seg loss bn_loss均为nan，想请问您一下是什么情况呢？

**Comment by Gchang9** [comment]
您好，由于我把近3000组图像划分为了训练、测试、验证。导致训练的时候最后一个batch不能整除。目前我把drop_last设置为True可以正常训练了。但Tensorboard的暂时还未发现解决方案、

**Comment by Gchang9** [comment]
Thank you, the loss function is nan because my batchsize is not divisible, I set Droplast to True and can train normally! There is actually a minor problem. I noticed that both the 'Models' folder and the main folder have a file named bisrnet.py, which defines the head layer, and the former does not. What is the purpose of these two? Thank you again!




在2022年5月23日 ***@***.***> 写道：

Hi. Is the batch size too small? The BiSRNet is a bit hard to train but the loss shouldn't be crazy.
I got accuracy improvements while freezing the other model parts and training only the SR modules. You can also try that.

**Comment by Joshua00044444** [comment]
请问是如何修改drop_last的参数的
